x-pack/plugins/security_solution/public/management/pages/policy/view/event_filters/layout/policy_event_filters_layout.tsx

```diff
@@ -56,6 +58,25 @@ export const PolicyEventFiltersLayout = React.memo<PolicyEventFiltersLayoutProps
       navigateCallback({ show: undefined });
     }, [navigateCallback]);
 
+    const assignToPolicyButton = useMemo(
```

Comment:
Could we also automatically close the flyout when the page is loaded with the flyout opened by url params and the license doesn't allow us to add event filters to the policy?

Reply:
Oh yes indeed. I forgot this! Will add it as well.

Reply:
done e393087a60272f07263e1682869f18ce048a72d8